sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -808,7 +810,9 @@ public Read<T> withCoder(Coder<T> coder) {
      * It should ONLY be used if the default value throws memory errors.
      */
     public Read<T> withFetchSize(int fetchSize) {
-      checkArgument(fetchSize > 0, "fetch size must be > 0");
+      checkArgument(
```

Comment:
```suggestion
      // Note that api.java.sql.Statement#setFetchSize says it only accepts values greater than >= 0 and that MySQL supports using Integer.MIN_VALUE as a hint to stream the ResultSet instead of loading it into memory. See https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-reference-implementation-notes.html for additional details.
      checkArgument(
```